yarn.lock

```diff
@@ -198,20 +198,20 @@ amdefine@>=0.0.4:
   resolved "https://registry.yarnpkg.com/amdefine/-/amdefine-1.0.1.tgz#4a5282ac164729e93619bcfd3ad151f817ce91f5"
   integrity sha1-SlKCrBZHKek2Gbz9OtFR+BfOkfU=
 
-angular-animate@1.4.4:
-  version "1.4.4"
-  resolved "https://registry.yarnpkg.com/angular-animate/-/angular-animate-1.4.4.tgz#70baec78ff49cba3be13ea0141819ffb2e785115"
-  integrity sha1-cLrseP9Jy6O+E+oBQYGf+y54URU=
+angular-animate@1.7.9:
```

Comment:
weird that lock file is also a part of git history.

Reply:
During the build process, `yarn install` is run with `frozen-version` flag. This must have something to do with this